metapool/sample_sheet.py

```diff
@@ -721,9 +730,39 @@ def quiet_validate_and_scrub_sample_sheet(self):
                               ' to be included in the Contact section.') %
                              ', '.join(sorted(contact - projects))))
 
+        # silently convert boolean values to either True or False and generate
+        # messages for all unrecognizable values.
+        if self.Bioinformatics is not None:
+            msgs += self._normalize_bi_booleans()
+
         # return all collected Messages, even if it's an empty list.
         return msgs
 
+    def _normalize_bi_booleans(self):
+        msgs = []
+
+        def func(x):
+            if type(x) is bool:
+                # column type is already correct.
+                return x
+            elif type(x) is str:
+                # strings should be converted to bool if possible.
+                if x.lower() == 'true':
+                    return True
+                elif x.lower() == 'false':
+                    return False
+
+            # if value isn't recognizably True or False, leave it
+            # unchanged and leave a message for the user.
+            msgs.append(f"'{x}' is not 'True' or 'False'")
+            return x
```

Comment:
I don't _think_ this branch is scoped in the unit tests, but it is plausible to encounter in production.

Reply:
It's not called directly in the unit tests, but it's called by KLSampleSheet.init() when new objects are initialized with a file path to a sample-sheet file. It's also called by quiet_validate_and_scrub_sample_sheet(), which is called by make_sample_sheet() when new objects are being created in a notebook, etc. Hence it ends up being called indirectly by many of the unit tests.